### DISCUSSION TITLE
statusline config

### DISCUSSION THREAD

**Discussion by notwidow** [famiu/feline.nvim#310] [General]
Hi, i want code for this config... 
![d](https://user-images.githubusercontent.com/112823418/190930671-f364caf7-0e65-4d1f-8ab7-6754f75416f5.png)
 i want  this exact config as i don't have knowledge about lua.
just replace the colors red yellow and 1 3 and arrows to UNIX.. thanks

**Comment by ram02z** [comment]
Hey,

The config was deleted from the user that submitted the theme. I found the config in one of the past commits here https://bitbucket.org/luizcoro/dotfiles/src/2bd13138baf5a86cc00c0fef048032313c4c1cf6/.config/nvim/lua/plugins/feline.lua

I haven't tested the following config and there might need to be changes since this commit is from 2021

**Reply by notwidow** [reply]
> Hey,
> 
> The config was deleted from the user that submitted the theme. I found the config in one of the past commits here https://bitbucket.org/luizcoro/dotfiles/src/2bd13138baf5a86cc00c0fef048032313c4c1cf6/.config/nvim/lua/plugins/feline.lua
> 
> I haven't tested the following config and there might need to be changes since this commit is from 2021

Hi, thanks for config but upon pasting the config .. the statusline show nothing ... i think changes have been made to feline and now need another config...
